tests/js/server/dump/dump-multiple.js

```diff
@@ -30,398 +30,86 @@
-      if (views.length === 0) {
-        return; // arangosearch views are not supported
-      }
-
-      let view = db._view("UnitTestsDumpView");
-      assertTrue(view !== null);
-      let props = view.properties();
-      assertEqual(Object.keys(props.links).length, 1);
-      assertTrue(props.hasOwnProperty("links"));
-      assertTrue(props.links.hasOwnProperty("UnitTestsDumpViewCollection"));
-      assertTrue(props.links.UnitTestsDumpViewCollection.hasOwnProperty("includeAllFields"));
-      assertTrue(props.links.UnitTestsDumpViewCollection.hasOwnProperty("fields"));
-      assertTrue(props.links.UnitTestsDumpViewCollection.includeAllFields);
-
-      assertEqual(props.consolidationIntervalMsec, 0);
-      assertEqual(props.cleanupIntervalStep, 456);
-      assertTrue(Math.abs(props.consolidationPolicy.threshold - 0.3) < 0.001);
-      assertEqual(props.consolidationPolicy.type, "bytes_accum");
-
-      var res = db._query("FOR doc IN " + view.name() + " SEARCH doc.value >= 0 OPTIONS { waitForSync: true } RETURN doc").toArray();
-      assertEqual(5000, res.length);
-
-      res = db._query("FOR doc IN " + view.name() + " SEARCH doc.value >= 2500 RETURN doc").toArray();
-      assertEqual(2500, res.length);
-
-      res = db._query("FOR doc IN " + view.name() + " SEARCH doc.value >= 5000 RETURN doc").toArray();
-      assertEqual(0, res.length);
-
-      res = db._query("FOR doc IN UnitTestsDumpView SEARCH PHRASE(doc.text, 'foxx jumps over', 'text_en')  RETURN doc").toArray();
-      assertEqual(1, res.length);
-    }
-
-  };
-}
-
-////////////////////////////////////////////////////////////////////////////////
-/// @brief executes the test suite
-////////////////////////////////////////////////////////////////////////////////
-
-jsunity.run(dumpTestSuite);
+const deriveTestSuite = require('@arangodb/test-helper').deriveTestSuite;
+const base = require("fs").join(
+  internal.pathForTesting('server'),
+  'dump',
+  'dump-test.inc');
+const baseTests = require("internal").load(base);
+
+
+jsunity.run(function dump_single_testsuite() {
+
+  let suite = {};
+
+  deriveTestSuite(
+    baseTests({
+    // testEmpty
+    emptyIndexes: 1,
+    // testMany
+    manyIndexes: 1,
+    manyCountStart: 0,
+    manyCount: 100000,
+    manyCountInc: 1,
+    // testEdges
+    edgesIndexCount: 2,
+    edgeIndexLoopCount: 10,
+    edgeIndexLoopInc: 1,
+    // testRemoved
+    removedCount: 9000,
+    removedSingleIndices: 1,
+    removedModulo: 10,
+    // testIndexes
+    indexesCount: 9,
+    // testKeygenAutoInc
+    keygenAutoInc: 42049,
+    autoIncDocCount: 1000, // this one is special
+    // testKeygenPadded
+    paddedDocCount: 1001,
+    // testKeygenUuid
+    uuidDocCount: 1001,
+  }),
+    suite,
+    "_multiple",
+    [
+      // no auth support here
+      "testUsers",
+      // cluster tests:
+      "testDatabaseProperties",
+      "testShards",
+      "testReplicationFactor",
+
+      "testKeygenPadded", // TODO: should we?
```

Comment:
TODO: should we?

Reply:
I don't know what that means. Either remove or describe so we know what to do here in the future.